aiogram/api/types/user.py

```diff
@@ -33,6 +41,80 @@ class User(TelegramObject):
 
     @property
     def full_name(self) -> str:
+        """
+        Get full name of user.
+        :return: str
+        """
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def mention(self) -> str:
+        """
+        Get user's username to mention him.
+        Full name will be returned if user has no username.
+        :return: str
+        """
+        if self.username:
+            return "@" + self.username
+        return self.full_name
+
+    @property
+    def locale(self) -> Optional[babel.core.Locale]:
+        """
+        Get user's locale
+        :return: :class:`babel.core.Locale`
+        """
+        if not self.language_code:
+            return None
+        if not hasattr(self, "_locale"):
```

Comment:
we're making `User` mutable and only for this? Can you consider using something like `cache_property`?

Reply:
Yes, it's just a simple port so far. This property actually requires discussion. Anyway, I don't like the idea of ​​leaving the object mutable.

Reply:
If we really need/want this method, I'd rename it to something different, like `babel_locale`